src/components/shared/Numeral/Numeral.tsx

```diff
@@ -43,18 +38,14 @@ const Numeral = ({
   prefix,
   suffix,
   className,
-  appearance,
   ...rest
 }: Props) => {
   const convertedValue = convertToDecimal(value, decimals || 0);
 
   const formattedValue = getFormattedNumeralValue(convertedValue, value);
 
   return (
-    <span
-      className={classNames(getMainClasses(appearance, styles), className)}
-      {...rest}
-    >
+    <span className={className} {...rest}>
```

Comment:
@mmioana, please be careful during rebase. I made changes to this file in the dashboard branch, so it will be a conflict 100% :)

Reply:
I'm preparing for some conflicts.. mentally 💯 thanks for the heads-up 🙌 ❤️